packages/sidenav/src/components/SideBar/SideBar.styled.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from "prop-types";
+import styled, { css } from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+import { FULLSCREEN, PUSH, OVERLAY } from "../../constants/variants";
+
+const fullScreenStyle = css`
+  top: 0;
+  ${({ expanded }) => (expanded ? "padding: 16px;" : "padding: 16px 0;")}
+  width: ${({ expanded }) => (expanded ? "100%" : "0px")};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  backdrop-filter: blur(5px);
+  opacity: 0.9;
```

Comment:
The filter doesn't work with opacity on safari, we need to use a semi-transparent background.